lib/ui/workoutPlayer/models/Exercise.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:html';
```

Comment:
Why are there 3 Exercise classes 😂?  There's one in `lib/models/exercise.dart`, one in `lib/ui/workoutPlayer/models/Exercise.dart`, and another one in `lib/ui/workoutPlayer/models/Exercise2.dart`. Are they all used? If not can we remove the unused ones?

Reply:
I think one was Andi's, Lance's, and mb Richard or Alec's? Make sure you keep the right one though!

Reply:
The one we want is the one that is in the `lib/models/exercise.dart`, we should edit all the code that uses the other two types of Exercises.

Reply:
we can't do that because for some reason whenever I utilise lance's version of exercises the video player does not show the list of exercise